static/js/containers/pages/DashboardPage.js

```diff
@@ -77,7 +77,13 @@ export class DashboardPage extends React.Component<DashboardPageProps, void> {
         <div className="dashboard container">
           <h1>My Courses</h1>
           <div className="enrolled-items container">
-            {enrollments && enrollments.map(this.renderEnrolledItemCard)}
+            {enrollments && enrollments.length > 0 ? (
+              enrollments.map(this.renderEnrolledItemCard)
+            ) : (
+              <div className="enrolled-item row card p-3 p-sm-5 rounded-0">
+                Once you enroll in a course, you can find it listed here.
```

Comment:
I like the phrase you added in the PR but just to mention, the message in the design is a little different which says `Once you enroll in a course, you can find the course listed here.`. Do you think we should match the design here?